public/stylesheets/sass/styles.scss

```diff
@@ -0,0 +1,29 @@
+body, html{
```

Comment:
Keep the style consistent, you've got 1 space between `{` and selectors in previous file and 0 spaces here